lib/galaxy/webapps/galaxy/buildapp.py

```diff
@@ -128,6 +129,9 @@ def paste_app_factory( global_conf, **kwargs ):
 
     register_postfork_function(postfork_setup)
 
+    for th in threading.enumerate():
+        if th.is_alive():
+            log.debug("Prior to webapp return, Galaxy thread %s is alive.", th)
```

Comment:
Hrmm.  I figured it'd be good to have a single picture of running threads in either context, but I'm fine with wrapping it too.